### PR TITLE
Adding additional test coverage for parsing

### PR DESCRIPTION
**Pull request by m-cheung**
- Additionally, the branch coverage requirement has been increased to 80% (Issue: #30)

### CODE REVIEW

**Comment by m-cheung** [comment]
Please hold off on merging for now. It seems like there is still a bug with recognizing `serialize;`

Additionally, I noticed that the test harness may not be working as intended (I believe it may not be performing the checks correctly)

File: config/istanbul.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@ check:
     statements: 85
     lines: 85
     # TODO(markdittmer): Write tests to bump branch threshold up to 80%.
```
Comment (mdittmer) [inline config/istanbul.yml]: Please delete TODO as well.

File: test/any/parsing/additional_idl_spec-helper.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Fragments of IDL scraped from various sources in May 2017. Stored in a js file for ease
+// of synchronous inclusion in diverse test environments.
+
+global.additional_idl_spec = `
```
Comment (mdittmer) [inline test/any/parsing/additional_idl_spec-helper.js]: nit: camelCase.

File: test/any/parsing/cmp-integration.js [hunk 3]
```diff
@@ -50,6 +50,11 @@ describe('Comparing parses', function() {
   );
 
   it(
+    'parse(spec) == parse(stringify(parse(spec)))',
```
Comment (mdittmer) [inline test/any/parsing/cmp-integration.js]: More descriptive string?

**Review by mdittmer — CHANGES_REQUESTED** [review]

**Review by mdittmer — APPROVED** [review]
LGTM

**Pull request merged**